qiskit/providers/aer/noise/errors/quantum_error.py

```diff
@@ -40,26 +53,29 @@ class QuantumError:
              module.
     """
 
-    # pylint: disable=invalid-name
-    _ATOL_DEFAULT = ATOL_DEFAULT
-    _RTOL_DEFAULT = RTOL_DEFAULT
-    _MAX_TOL = 1e-4
-
     def __init__(self,
-                 noise_ops,
+                 noise_ops: Union[Union[BaseOperator,
+                                        QuantumCircuit,
+                                        List[Tuple[Instruction, List[int]]],
+                                        Tuple[Instruction, List[int]],
+                                        Instruction],
+                                  Iterable[Tuple[Union[BaseOperator,
+                                                       QuantumCircuit,
+                                                       List[Tuple[Instruction, List[int]]],
+                                                       Tuple[Instruction, List[int]],
+                                                       Instruction],
+                                                 float]]],
                  number_of_qubits=None,
-                 standard_gates=True,
-                 atol=ATOL_DEFAULT):
+                 standard_gates=False,
```

Comment:
In the new input interface, I think we don't need to support `standard_gates` functionality, if a `UnitaryGate` is supplied, it remains `UnitaryGate` even if it has a "standard gate" matrix, e.g. X gate matrix. This is the biggest change in this noise module refactoring. I think it does not affect users so much because most of them would not care about how quantum errors are represented internally.